system-tests/tests/smoke/cre/por_test.go

```diff
@@ -334,7 +334,73 @@ func configureDataFeedsCacheContract(testLogger zerolog.Logger, input *configure
 	return configErr
 }
 
-func registerPoRWorkflow(input registerPoRWorkflowInput) error {
+func pausePoRWorkflow(input managePoRWorkflowInput) error {
+	// This env var is required by the CRE CLI
+	err := os.Setenv("CRE_ETH_PRIVATE_KEY", input.deployerPrivateKey)
+	if err != nil {
+		return errors.Wrap(err, "failed to set CRE_ETH_PRIVATE_KEY")
+	}
+
+	workflowRegistryAddress, workflowRegistryErr := crecontracts.FindAddressesForChain(input.addressBook, input.homeChainSelector, keystone_changeset.WorkflowRegistry.String())
+	if workflowRegistryErr != nil {
+		return errors.Wrapf(workflowRegistryErr, "failed to find workflow registry address for chain %d", input.homeChainSelector)
+	}
+
+	pauseWorkflowInput := keystonetypes.ManageWorkflowWithCRECLIInput{
+		ChainSelector:            input.chainSelector,
+		WorkflowDonID:            input.workflowDonID,
+		WorkflowRegistryAddress:  workflowRegistryAddress,
+		WorkflowOwnerAddress:     input.sethClient.MustGetRootKeyAddress(),
+		CRECLIPrivateKey:         input.deployerPrivateKey,
+		CRECLIAbsPath:            input.creCLIAbsPath,
+		CRESettingsFile:          input.creCLIsettingsFile,
+		WorkflowName:             input.WorkflowConfig.WorkflowName,
+		ShouldCompileNewWorkflow: input.WorkflowConfig.ShouldCompileNewWorkflow,
+		CRECLIProfile:            input.creCLIProfile,
+	}
+
+	pauseErr := creworkflow.PauseWithCRECLI(pauseWorkflowInput)
+	if pauseErr != nil {
+		return errors.Wrap(pauseErr, "failed to pause workflow with CRE CLI")
+	}
+
+	return nil
+}
+
+func activatePoRWorkflow(input managePoRWorkflowInput) error {
+	// This env var is required by the CRE CLI
+	err := os.Setenv("CRE_ETH_PRIVATE_KEY", input.deployerPrivateKey)
+	if err != nil {
+		return errors.Wrap(err, "failed to set CRE_ETH_PRIVATE_KEY")
+	}
+
+	workflowRegistryAddress, workflowRegistryErr := crecontracts.FindAddressesForChain(input.addressBook, input.homeChainSelector, keystone_changeset.WorkflowRegistry.String())
+	if workflowRegistryErr != nil {
+		return errors.Wrapf(workflowRegistryErr, "failed to find workflow registry address for chain %d", input.homeChainSelector)
+	}
+
+	activateWorkflowInput := keystonetypes.ManageWorkflowWithCRECLIInput{
+		ChainSelector:            input.chainSelector,
+		WorkflowDonID:            input.workflowDonID,
+		WorkflowRegistryAddress:  workflowRegistryAddress,
+		WorkflowOwnerAddress:     input.sethClient.MustGetRootKeyAddress(),
+		CRECLIPrivateKey:         input.deployerPrivateKey,
+		CRECLIAbsPath:            input.creCLIAbsPath,
+		CRESettingsFile:          input.creCLIsettingsFile,
+		WorkflowName:             input.WorkflowConfig.WorkflowName,
+		ShouldCompileNewWorkflow: input.WorkflowConfig.ShouldCompileNewWorkflow,
+		CRECLIProfile:            input.creCLIProfile,
+	}
+
+	activateErr := creworkflow.ActivateWithCRECLI(activateWorkflowInput)
+	if activateErr != nil {
+		return errors.Wrap(activateErr, "failed to activate workflow with CRE CLI")
+	}
+
+	return nil
```

Comment:
also, that method if basically identical to `pausePoRWorkflow`, can we extract the common part?